### PR TITLE
Initial support for JP5

### PR DESCRIPTION
**Pull request by ricardosalveti**
Still not using systemd-boot by default, but required work for enabling it is also part of this PR.

### CODE REVIEW

File: meta-lmp-base/conf/distro/include/non-clangable.inc [hunk 1]
```diff
@@ -99,6 +99,9 @@ TOOLCHAIN:pn-trusted-firmware-a = "gcc"
 
 # jetson-agx-xavier-devkit
 TOOLCHAIN:pn-cboot-t19x = "gcc"
```
Comment (quaresmajose) [inline meta-lmp-base/conf/distro/include/non-clangable.inc:101]: as cboot is been dropped we can drop this as well

**Review by quaresmajose — APPROVED** [review]
LGTM with a minor comment

**Review by MrCry0 — APPROVED** [review]
LGTM. (with jose's note)

File: meta-lmp-base/conf/distro/include/non-clangable.inc (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-base/conf/distro/include/non-clangable.inc:101]: yup, will remove.

Comment (ricardosalveti) [reply meta-lmp-base/conf/distro/include/non-clangable.inc]: done

**Review by quaresmajose — APPROVED** [review]
LGTM

**Pull request merged**